# CVE-2015-3206


## Metadata
| Published | Last Modified | Assigner | Version | Format |
| --------- | ------------- | -------- | ------- | ------ |
| 2017-08-25 18:29:00 | 2018-12-20 18:11:00 | secalert@redhat.com | 4.0 | MITRE | 

## Description
| Language | Description |
| -------- | ----------- |
| en | The checkPassword function in python-kerberos does not authenticate the KDC it attempts to communicate with, which allows remote attackers to cause a denial of service (bad response), or have other unspecified impact by performing a man-in-the-middle attack. |

## Configurations
| Vulnerable | CPE 23 URI | CPE Name | Operator |
| ---------- | ---------- | -------- | -------- |
| True | cpe:2.3:a:apple:pykerberos:-:*:*:*:*:*:*:* | [] | OR |

## CVSSv3 Information
| Score | Severity | Impact Score | Exploitability Score | Vector String | Version | Attack Vector | Attack Complexity | Privileges Required | User Interaction | Scope | Confidentiality Impact | Integrity Impact | Availability Impact |
| ----- | -------- | ------------ | -------------------- | ------------- | ------- | ------------- | ----------------- | ------------------- | ---------------- | ----- | ---------------------- | ---------------- | ------------------- |
| 8.1 | HIGH | 5.9 | 2.2 | CVSS:3.0/AV:N/AC:H/PR:N/UI:N/S:U/C:H/I:H/A:H | 3.0 | NETWORK | HIGH | NONE | NONE | UNCHANGED | HIGH | HIGH | HIGH |

## CVSSv2 Information
| Score | Severity | Impact Score | Exploitability Score | Vector String | Version | Access Vector | Access Complexity | Authentication | User Interaction Required | Obtain All Privilge | Obtain User Privilege | Obtain Other Privilege | Confidentiality Impact | Integrity Impact | Availability Impact |
| ----- | -------- | ------------ | -------------------- | ------------- | ------- | ------------- | ----------------- | -------------- | ------------------------- | ------------------- | --------------------- | ---------------------- | ---------------------- | ---------------- | ------------------- |
| 6.8 | MEDIUM | 6.4 | 8.6 | AV:N/AC:M/Au:N/C:P/I:P/A:P | 2.0 | NETWORK | MEDIUM | NONE | 0 | False | False | False | PARTIAL | PARTIAL | PARTIAL |

## Problem Type
CWE-287

## References
| Name | Reference URL | Reference Source | Tags |
| ---- | ------------- | ---------------- | ---- |
https://pypi.python.org/pypi/kerberos|https://pypi.python.org/pypi/kerberos|CONFIRM|['Patch', 'Vendor Advisory']
| Name | Reference URL | Reference Source | Tags |
| ---- | ------------- | ---------------- | ---- |
https://github.com/apple/ccs-pykerberos/issues/31|https://github.com/apple/ccs-pykerberos/issues/31|CONFIRM|['Issue Tracking', 'Patch', 'Third Party Advisory']
| Name | Reference URL | Reference Source | Tags |
| ---- | ------------- | ---------------- | ---- |
https://bugzilla.redhat.com/show_bug.cgi?id=1223802|https://bugzilla.redhat.com/show_bug.cgi?id=1223802|CONFIRM|['Issue Tracking', 'Patch', 'Third Party Advisory']
| Name | Reference URL | Reference Source | Tags |
| ---- | ------------- | ---------------- | ---- |
74760|http://www.securityfocus.com/bid/74760|BID|['Third Party Advisory', 'VDB Entry']
| Name | Reference URL | Reference Source | Tags |
| ---- | ------------- | ---------------- | ---- |
[oss-security] 20150521 CVE-2015-3206 python-kerberos: checkPassword() does not verify KDC authenticity|http://www.openwall.com/lists/oss-security/2015/05/21/3|MLIST|['Mailing List', 'Patch', 'Third Party Advisory']
